### PR TITLE
Contacts endpoint

### LINKED ISSUE #20: Add support for contacts
As mentioned in [Platform-Aware @-mentioning People on my Blog](https://www.jvt.me/posts/2020/03/22/at-mention-people/) I found that setting up the `q=contact` Micropub query like so:

```json
{
  "contacts": [
    {
      "name": "Barry Frost",
      "nickname": "barryfrost.com",
      "url": "https://barryfrost.com/",
      "silos": {
        "twitter": "BarryF"
      },
      "_internal_url": "https://www.jvt.me/contacts/barryfrost.com/"
    }
  ]
}
```

Makes it handy to perform different syndication, as well as having autocomplete of people in i.e. Indigenous for Android.

### PR DESCRIPTION
**Pull request by barryf**
- Sandbox dummy data
- Table for contacts
- Filtering support
- Endpoint for ?q=contact&filter=x

Closes #20

### CODE REVIEW

File: src/http/get-micropub/config/contact.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }
+  const data = await arc.tables()
+  const results = await data.contacts.scan({})
+  const contacts = results.Items.map(c => {
+    if (filter) {
+      if (c.nickname.startsWith(filter)) {
+        return c
+      } else {
+        return null
+      }
```
Comment (jamietanna) [inline src/http/get-micropub/config/contact.js:15]: This shouldn't be needed, as it'd return `null` / `undefined` anyway?

```suggestion
```

**Comment by jamietanna** [comment]
Awesome, thanks :raised_hands: We'd need to hook this in with CRUD, too, I'd say

File: src/http/get-micropub/config/contact.js (hunk 1 above)
Comment (barryf) [reply src/http/get-micropub/config/contact.js:15]: I get the following from Standard.js without returning the null: 
> Array.prototype.map() expects a value to be returned at the end of arrow function. (array-callback-return)

**Comment by barryf** [comment]
Releasing. NB: #22 is for adding the Micropub actions.

File: src/http/get-micropub/config/contact.js (hunk 1 above)
Comment (jamietanna) [reply src/http/get-micropub/config/contact.js:15]: Interesting - fair enough!

**Pull request merged**